README.md

```diff
@@ -25,6 +25,12 @@ or
 npm install tailwindcss-stimulus-components
 ```
 
+or
```

Comment:
This library can be used without Rails and these instructions are Rails specific.

```suggestion
or use with Rails importmaps:
```